api/routes/me/put.js

```diff
@@ -39,7 +39,7 @@ module.exports = (app, { deserialize = deserializeUser } = {}) => {
       } catch (e) {
         res
           .status(400)
-          .send({ error: `update-self-${e.message}` })
+          .send({ error: `edit-self.${e.message}` })
```

Comment:
Using a different delimiter between the two parts of the error message/token/whatever lets us start to group the messages logically.  This comes up further down in the PR.

I only changed the error tokens used by this PR, since they're the only ones actually being consumed.  We can update the others as we use them.  There shouldn't be too many of them.